### DISCUSSION TITLE
How to export the generated GraalVM native image

### DISCUSSION THREAD

**Discussion by helpermethod** [jbangdev/jbang#1311] [General]
Hi,

I have a nice little JBang script which I would like to distribute as a native binary via JReleaser.

Building the native image works fine locally, but I wonder to export it. `jbang export` currently only seems to work with the generated JAR file?

**Comment by maxandersen** [comment]
jbang export definitely should work with native too. 

'jbang --native export local' ?

**Reply by helpermethod** [reply]
I seem to be doing something wrong:

`jbang --native export local i-can-haz-sdk.java`

<img width="508" alt="image" src="https://user-images.githubusercontent.com/1562019/158545886-d6656b00-c011-44ca-aec6-e7b70489c90a.png">

**Reply by helpermethod** [reply]
Exporting the JAR works

<img width="711" alt="image" src="https://user-images.githubusercontent.com/1562019/158546048-f5da3ef2-21dd-4152-9d26-b819b21b0e27.png">

**Reply by helpermethod** [reply]
Ah, this seems to work

`jbang export local --native i-can-haz-sdk.java`

**Reply by maxandersen** [reply]
Sorry. Was just on mobile couldn't verify :)
